ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2659,3 +2661,47 @@ def test044_table_with_add_edit_single_row(self):
 
         self.info('Assert that the test unit not equal ')
         self.assertNotEqual(testunit_before_edit_row, testunit_after_edit_row)
+
+    def test047_upload_attachment(self):
+        """
+        I can upload any attachment successfully from the order section
+        LIMS-8258
+        :return:
+        """
+        # get random order
+        order, payload = self.orders_api.create_new_order()
+        self.orders_page.get_order_edit_page_by_id(id=order['order']['mainOrderId'])
+        self.base_selenium.click(element='order:attachment_btn')
+        self.info('this is the file name from the assets ')
+        file_name = 'logo.png'
+        upload_file = self.order_page.upload_attachment(file_name='logo.png', drop_zone_element='order:uploader_zone', save=True)
+        self.info("upload file then press on the save button it will return the file name ".format(upload_file))
+        self.info("assert that the upload file same as the file name ".format(upload_file, file_name))
+        self.assertEqual(upload_file, file_name)
+
+    @skip('https://modeso.atlassian.net/browse/LIMS-177')
+    def test048_upload_attachment_then_remove(self):
+        """
+        Orders step 1: Attachment download approach: There is a link under remove link for
+        download and you can preview it by clicking on it
+        LIMS-6933
+        :return:
+        """
+        # get random order
+        order, payload = self.orders_api.create_new_order()
+        self.orders_page.get_order_edit_page_by_id(id=order['order']['mainOrderId'])
+        self.base_selenium.click(element='order:attachment_btn')
+        self.info('this is the file name from the assets ')
+        file_name = 'logo.png'
+        upload_attachment_then_save = self.order_page.upload_attachment(file_name='logo.png', drop_zone_element='order:uploader_zone', save=True)
+        self.info("upload file to enter to the same record to remove this file ".format(upload_attachment_then_save))
+        self.info("assert that the ")
+        self.assertEqual(upload_attachment_then_save, file_name)
+        self.info('oprn the same record in the edit mode')
+        self.orders_page.get_order_edit_page_by_id(id=order['order']['mainOrderId'])
+        self.base_selenium.click(element='order:attachments_btn')
+        after_remove_attachment = self.order_page.upload_attachment(file_name='logo.png',drop_zone_element='order:uploader_zone', remove_current_file=True,save=True)
+        self.info("remove the file and submit the record ".format(after_remove_attachment))
```

Comment:
u need to upload `logo2.png` here instead of uploading the same existing file and this log should be before the uploading step not after it.

Reply:
Done